src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/negationRationale/NegationRationale.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState } from "react";
+import "twin.macro";
+import "styled-components/macro";
+import { IconButton, Chip } from "@mui/material";
+import AddCircleOutlineIcon from "@mui/icons-material/AddCircleOutline";
+import _ from "lodash";
+import { DataElement, CqmMeasure, CQL } from "cqm-models";
+import { makeStyles } from "@mui/styles";
+import CodeInput from "../../../../../../../common/codeInput/CodeInput";
+
+interface Chip {
```

Comment:
is this being used? also, this seems a name collision with the Chip that you are importing from mui.